src/pages/VenuePage/VenuePage.tsx

```diff
@@ -265,12 +263,12 @@ export const VenuePage: React.FC = () => {
   }
 
   if (profile && !isCompleteProfile(profile)) {
-    return <Redirect to={`/account/profile?venueId=${venueId}`} />;
+    return <Redirect to={`/account/profile?spaceSlug=${spaceSlug}`} />;
```

Comment:
[NOTE]
Should be using `ACCOUNT_PROFILE_VENUE_PARAM_URL` from `urlSettings.ts` through `accountProfileVenueUrl()` from `url.ts`